apps/api/src/routes/legacy_apis_deleteApi.ts

```diff
@@ -6,7 +6,7 @@ import { rootKeyAuth } from "@/pkg/auth/root_key";
 import { UnkeyApiError, openApiErrorResponses } from "@/pkg/errors";
 import { schema } from "@unkey/db";
 import { newId } from "@unkey/id";
-import { buildQuery } from "@unkey/rbac";
```

Comment:
this is typesafe now

Reply:
specifically typesafe for our own permissions

it's still generic and we will use the `buildQuery` for our sdk